client/src/components/Home.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Home = () => (
+  <div>
+    <h1>enter an id from 1 - 100</h1>
+  </div>
+)
+
+export default Home;
```

Comment:
I like this as a testing option, though I think Leslie may ask you to change it to pulling from the url.